kornia/color/core.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union, Tuple
+
+import torch
+import torch.nn as nn
+
+
+def add_weighted(src1: torch.Tensor, alpha: float,
+                 src2: torch.Tensor, beta: float,
+                 gamma: float) -> torch.Tensor:
+    r"""Blend two Tensors.
+
+    See :class:`~kornia.color.AddWeighted` for details.
+    """
+    if not isinstance(src1, torch.Tensor):
+        raise TypeError("src1 should be a tensor. Got {}".format(type(src1)))
+
+    if not isinstance(src2, torch.Tensor):
+        raise TypeError("src2 should be a tensor. Got {}".format(type(src2)))
+
+    if not isinstance(alpha, float):
+        raise TypeError("alpha should be a tensor or float. Got {}".format(type(alpha)))
```

Comment:
The type hint in line 9 says `alpha`, `beta`, `gamma` are `float`s, and we are checking that it is indeed `float`, but the error message says `tensor` or `float`
Maybe you want to use `typing.Union` in the type hint if you want to to accept `tensor` and `float` and also extend the type check in line 20 to accept `tensor`
or
maybe you need to correct the error message to remove `tensor` from the error string

Reply:
Ah. Thanks. I previously intended it to be `Tensor` or `float` but then planned against it.